WebServices/AzureNotificationHub/NotificationHubSample/NotificationHubSample.Android/FirebaseService.cs

```diff
@@ -62,5 +72,25 @@ void SendMessageToMainPage(string body)
         {
             (App.Current.MainPage as MainPage)?.AddMessage(body);
         }
+
+        void SendRegistrationToServer(string token)
+        {
+            try
+            {
+                NotificationHub hub = new NotificationHub(AppConstants.NotificationHubName, AppConstants.ListenConnectionString, this);
+
+                // register device with Azure Notification Hub using the token from FCM
+                Registration reg = hub.Register(token, AppConstants.SubscriptionTags);
+
+                // subscribe to the SubscriptionTags list with a simple template.
+                string pnsHandle = reg.PNSHandle;
+                var cats = string.Join(", ", reg.Tags);
+                var temp = hub.RegisterTemplate(pnsHandle, "defaultTemplate", AppConstants.FCMTemplateBody, AppConstants.SubscriptionTags);
```

Comment:
I'd recommend declaring temp as the type returned from the RegisterTemplate method, as it's not obvious from the code what that type is.